sw/device/lib/testing/meson.build

```diff
@@ -15,6 +15,25 @@ sw_lib_testing_test_status = declare_dependency(
   )
 )
 
+# Test coverage library that provides runtime functions for LLVM profiling.
+sw_lib_testing_test_coverage = declare_dependency(
+  link_with: static_library(
+    'test_coverage',
+    include_directories: sw_vendor_llvm_clang_rt_inc_dir,
+    sources: [sw_vendor_llvm_clang_rt_sources,
+      files('test_coverage.c'),
+    ],
+    dependencies: [
+      sw_lib_mem,
+      sw_lib_uart,
+    ],
+  ),
+)
+# Compile arguments to enable coverage.
+compile_args_for_coverage = declare_dependency(
```

Comment:
Nit: Given that this is an opaque dependency, why don't we just name it `enable_coverage`?

Reply:
Sounds good to me. Done.